app/src/main/java/no/usn/mob3000/ui/screens/info/InfoScreen.kt

```diff
@@ -77,9 +77,9 @@ private fun InfoButton(
 ) {
     Button(
         onClick = onClick,
-        colors = ButtonDefaults.buttonColors(containerColor = DefaultButton),
+        colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
         modifier = Modifier
             .fillMaxWidth()
             .height(56.dp)
-    ) { Text(text) }
+    ) { Text(text,) }
```

Comment:
There should not be a comma in when using only a singular parameter. Should be `Text(text)`.